CodeForAfricaLabs/CitizenReporter.Android
# Fix save story

## What does this PR do?

This PR fixes the save Story bug such that you don't have multiple empty Stories appearing in the Stories View. It also allows you to see the correct file size for recordings attached to the stories

## Description of Task to be completed?

Fix the save Story feature.

## How should this be manually tested?

- Open the Assignment Detail Page
- Open the Storyboard Activity and close it immediately by pressing the `Back` button
- Click on the Stories button in the bottom Navigation window
- There shouldn't be a story added other than the ones pulled from the server
- Repeat steps 1 and this time add a title to your story
- This time there should be a story added to the Stories Recycler View
- Also record an Audio and click the tick to attach depending on the length there should be an accurate FileSize value in **KBs** or **MBs**, depending on the size.
